src/app/core/data/metadata-value-data.service.ts

```diff
@@ -59,22 +59,22 @@ export class MetadataValueDataService extends BaseDataService<MetadataValue> imp
    * Retrieve the MetadataValue object inside Vocabulary object body
    */
   findByMetadataNameAndByValue(metadataName, term = ''): Observable<PaginatedList<MetadataValue>> {
-      const metadataFields = metadataName.split('.');
+      const metadataFields = metadataName?.split('.');
 
       const schemaRP = new RequestParam('schema', '');
       const elementRP = new RequestParam('element', '');
       const qualifierRP = new RequestParam('qualifier', '');
       const termRP = new RequestParam('searchValue', term);
 
       // schema and element are mandatory - cannot be empty
-      if (!isNotEmpty(metadataFields[0]) && !isNotEmpty(metadataFields[1])) {
+      if (!isNotEmpty(metadataFields?.[0]) && !isNotEmpty(metadataFields?.[1])) {
```

Comment:
the more negation the better?
! not
!! not not ;)

Reply:
Fixed, thank you